src/main/play-doc/operation/Monitoring.md

```diff
@@ -10,7 +10,11 @@ ConductR is a Lightbend Platform based application and has been configured to su
 
 ### Enabling ConductR Monitoring for Takipi
```

Comment:
I've recently changed this page for 2.1 given the OpsClarity integration. This will not be a straight cherry pick.